sdk/data/azcosmos/cosmos_container.go

```diff
@@ -25,6 +25,20 @@ type ContainerClient struct {
 	link string
 }
 
+// ItemIdentity represents the identity of an item (its id plus its partition key value).
+// This is useful for bulk/read-many style operations that need to address multiple
+// items under (potentially) different partition key values.
+//
+// ID must match the 'id' property of the stored item. PartitionKey is the value (or
+// composite/hierarchical set of values) the item was written with. For hierarchical
+// partition keys create the PartitionKey with NewPartitionKey* helpers (e.g.
+// NewPartitionKeyString, NewPartitionKeyInt, or NewPartitionKeyArray) following the
+// order defined in the container.
```

Comment:
nit: Perhaps a note about how you must specify _all_ levels of the partition key in an HPK scenario? I've seen that trip some users up.